mne_connectivity/utils/utils.py

```diff
@@ -71,6 +72,47 @@ def check_indices(indices):
     return indices
 
 
+def check_multivariate_indices(indices):
+    """Check indices parameter for multivariate connectivity and pad it.
```

Comment:
I know the other function, `check_indices` is not well documented necessarily either, but maybe we can start now :p

E.g. What does it check for?